DscResources/MSFT_xWindowsUpdateAgent/MSFT_xWindowsUpdateAgent.psm1

```diff
@@ -321,7 +322,7 @@ function Get-TargetResource
         [System.String[]]
         $Category= @('Security'),
         
-        [ValidateSet("Disabled","ScheduledInstallation")]
+        [ValidateSet("Disabled","ScheduledInstallation","Scheduled installation")]
```

Comment:
Instead of adding this (and forgetting to remove the wrong one leaving a bug) it's probably better to change line 216 to have the value in the external schema.